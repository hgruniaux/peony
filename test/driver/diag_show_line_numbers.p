//# RUN: peony -fdiagnostics-show-line-numbers %s
//# RUN: peony -fno-diagnostics-show-line-numbers %s

fn test() {
    5 + true;
    //~^ ERROR: cannot add 'i32' to 'bool'
}
