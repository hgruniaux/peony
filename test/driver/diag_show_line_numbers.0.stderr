<source>:5:7: error: cannot add 'i32' to 'bool'
    5 |     5 + true;
      |     ~ ^ ~~~~
